### PR TITLE
Failure highlight

### PR DESCRIPTION
**Pull request by nato000**

### CODE REVIEW

File: games/lockpicking/src/js/game/Game.js [hunk 1]
```diff
@@ -171,6 +176,15 @@ export class Game {
...
       throw new Error();
+    } else if (event.keyCode === 32) {
+      document
+        .querySelector(".unlock-label__img")
+        .classList.add("unlock-label__img--active");
+      setTimeout(() => {
+        document
+          .querySelector(".unlock-label__img")
+          .classList.remove("unlock-label__img--active");
+      }, 100);
```
Comment (frontend-sensei) [inline games/lockpicking/src/js/game/Game.js:187]: This method should not be changed.
Remove this logic from method.

File: games/lockpicking/src/js/game/Game.js [hunk 2]
```diff
@@ -140,6 +140,11 @@ export class Game {
 
       const positionCorrect = this._coordinates.checkPosition();
       if (!positionCorrect) {
+        document.querySelector(".bar").classList.add("bar--failure");
```
Comment (frontend-sensei) [inline games/lockpicking/src/js/game/Game.js:143]: Save bar node to the variable

File: games/lockpicking/src/js/game/Game.js [hunk 3]
```diff
@@ -140,6 +140,12 @@ export class Game {
 
       const positionCorrect = this._coordinates.checkPosition();
       if (!positionCorrect) {
+        const bar = document.querySelector(".bar");
+        bar.classList.add("bar--failure");
+        setTimeout(() => {
+          bar.classList.remove("bar--failure");
+        }, 1500);
```
Comment (frontend-sensei) [inline games/lockpicking/src/js/game/Game.js:147]: Create timeout const and use everywhere

File: games/lockpicking/src/styles/game/bar.css [hunk 4]
```diff
@@ -22,17 +22,12 @@
...
-  box-shadow: inset 0 0 0px 1px #000;
-}
-
 .bar-body__side-back {
   height: var(--body-length);
   width: var(--body-width);
   top: 50%;
-  transform: translateZ(calc(var(--area-width) * -0.5)) translateY(-50%);
+  transform: translateZ(calc(var(--area-width) * -0.5)) translateY(-50%)
+    rotateY(180deg);
```
Comment (nato000) [inline games/lockpicking/src/styles/game/bar.css:30]: image

image
(0deg)

**Review by frontend-sensei — APPROVED** [review]

**Pull request merged**